spec/decorators/usps_decorator_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+class DummyController < ApplicationController
+  def initialize
```

Comment:
@zachmargolis how do I reference `session` without access to the controller though?